diagnostic.md

```diff
@@ -73,13 +94,26 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request "POST" "http://www.example.com/donuts" \
+  --header "Content-Type: application/json" \
+  --data $""{
+    \"donut\": {
+      \"name\": "French Cruller",
+      \"price\": "$0.99"
+    }
+  }
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function () {$.ajax({
+    url: 'http://www.example.com/donuts/',
+    method: 'POST',
```

Comment:
You need the `data` on a `POST`: It would be something like:

```js
data: {
     "donut": {
           "name": "French Cruller",
           "price": "$0.99",
        }
     }
};
```